src/utils.py

```diff
@@ -29,28 +37,71 @@ def format_token_short(token):
 
 
 def format_integer(number):
-  return str(number) # TODO: Format better the numbers
+  return '{:,d}'.format(number)
```

Comment:
Nit: you can also use f strings for that https://stackoverflow.com/a/52650045/1272513

Reply:
Great tip!!!

https://github.com/gnosis/dex-cli/issues/39